### ISSUE TITLE
Method to exit edit mode is unintuitive

### ISSUE THREAD

**Issue by jameshfisher** [openstreetmap/openstreetmap-website#737]
After posting [a bug report on the iD project](https://github.com/openstreetmap/iD/issues/2216), @tyrasd informed me that this would be a better place.

As you'll see, I was unable to find how to exit edit mode. I'm now aware that I can "click the logo", but I'm suggesting that that isn't a good solution.

By having an "Edit" button while in "View" mode, and the same "Edit" button bolded while in "Edit" mode, the current UI tells me that the "Edit" button is something like a toggle button: I can click it to toggle in and out of edit mode.

When I saw that this wasn't the case, I then looked for a "close"/"cross" button in a corner, based on the intuition that I had been "dropped into" edit mode, and so I was looking for a way to "escape" it.

When I saw that this also wasn't the case, I gave up. and examined the URLs in order to manually construct it.

Since this process was entirely driven by web conventions/intuitions I've built up over time, I expect that an average web user would follow a similar process and get slightly frustrated like I did.

Normally, "clicking the logo" takes you to the homepage of whatever site you're on. The behavior I expected in the context of OSM is that the logo would take me to `http://www.openstreetmap.org/`, not to `http://www.openstreetmap.org/#map=A/B/C`. The intuition here is that the logo usually acts like a "reset" button.

IMO, an explicit "exit edit mode" button would be more intuitive.

I wonder whether anyone has done any [hallway testing](https://en.wikipedia.org/wiki/Usability_testing#Hallway_testing) on this?

**Comment by 123MT** [comment]
I encountered the same frustration and had to google "osm id exit edit" to determine how to exit edit mode.  Why not a "view" button that replaces the edit button+dropdown after edit mode is entered? This would follow same convention as the side menus which toggle menus on and off with subsequent clicks of the icon.  I agree fully with the original poster that this is a very frustrating portion of the UI of an otherwise intuitive map editor.

I will now click on the logo to exit edit mode which makes no sense to me or anyone else except perhaps the original developer.

**Comment by pgirolami** [comment]
Googling "osm how to exit edit mode in id" because I was stuck too brought me here... that must mean something...

**Comment by timmc** [comment]
@tomhughes I think it's pretty clear from this thread that the current user interface is confusing. Clicking the logo of a site *always* means "go back to the home page, lose all navigation state".

Would you accept a PR adding a View tab?

**Comment by GuidoS** [comment]
I support the idea of adding 'end edit session'  to the dropdown menu on the edit button.

**Comment by jidanni** [comment]
Yes I have several college degrees and I couldn't figure it out either.

User clicks "edit", arriving him at e.g.,
http://www.openstreetmap.org/edit#map=16/24.1640/120.8191 .

Sooner or later when the user wants to leave edit mode and return to
normal ( http://www.openstreetmap.org/#map=16/24.1640/120.8191 ) mode...

**Comment by richlv** [comment]
i've mapped for quite a few years with various editors, tested iD when it was just unveiled and used it a bit. using josm mostly. (this isn't bragging, just to explain that existing users are struggling with this).

this is the first time ever i see that clicking the logo is supposed to exit the edit mode. that's a very well hidden easter egg :)

i recently spoke with another contributor in an osm meetup - they have done a lot of mapping, too, and mostly with iD. somebody new asked them "oh, how do you go back to the map ?". the experienced iD user just shrugged and said "i just always go back in the browser history".

another weirdness goes like this : i have josm set as the default editor. if i open iD, then click 'edit' again, it goes back to the map, URL has "edit#map", and it tries to edit in josm. quite confusing.

i'd propose to change the "Edit" label and operation to be something like "Exit edit mode", or "Back to map". that would be technically correct and understandable for new users.

**Comment by mcepl** [comment]
Before I open new ticket (possibly duplicate), let me ask here as it is a bit related: a very natural thing after I found some place is to ask for the route to it. So, I would expect that if after finding a place I click on the "Directions" button, it would be automatically selected as a target of new route. Is there a ticket for this functionality?

**Comment by tomhughes** [comment]
I'm not sure what you mean by "finding a place" or what it has to do with an iD related ticket... In general there are tickets about better integration of the routing though, mostly about being able to right click on the map I think.

**Comment by mcepl** [comment]
oh, shut ... I missed that although it is on ``openstreetmap-website`` repo, it is actually really iD. Sorry.

**Comment by mwtoews** [comment]
Jumping in here to say that I found exiting the Edit mode confusing. I was trying to click "Edit" to turn it off, or finding something in the drop down. However, thanks to this thread, I see how to exit: click the logo. I agree, this is not an intuitive way to toggle the edit mode.

**Comment by meskobalazs** [comment]
Sorry for the thread necromancy, but I had to do an online search to exit the edit mode. I found this issue in the results. This reminds me of vi :)

**Comment by danieldegroot2** [comment]
Sidenote: a button to solely 'Discard all changes', which would be in addition not as a replacement, is being discussed at [iD#10045](https://github.com/openstreetmap/iD/pull/10045)
